gke-dws-examples/manifests.yaml

```diff
@@ -0,0 +1,11687 @@
+apiVersion: v1
```

Comment:
This installs a bunch of Kueue CRDs, are you sure you want this here?

Reply:
@mwielgus was is different here from the regular installation?
Is it just the feature gate?

Could we replace this with a script that:
1. downloads the installation manifest
2. adds the feature gate
3. `apply --server-side`

Reply:
The problem with the kueue manifest is that it is 10k+ lines long. We don't want clients to edit this file as a starter.

Reply:
Users may be reluctant to use scripts that do some magic they cannot check/control. Manifests, white big and cumbersome, seem simpler and easier to adopt.